azure/azure-release-pipeline.yml

```diff
@@ -30,6 +30,7 @@ extends:
     service_name: ${{ variables.service_name }}
     short_service_name: ${{ variables.short_service_name }}
     service_base_path: ${{ variables.service_base_path }}
+    prod_requires_approval: false
```

Comment:
We want Prod to be manually approved before release so I imagine this needs to be 'true'